extruct/uniform.py

```diff
@@ -4,7 +4,13 @@
 def _uopengraph(extracted):
     out = []
     for obj in extracted:
-        flattened = dict(reversed(obj['properties']))
+        # In order of appearance in the page
+        properties = list(reversed(obj['properties']))
+        # Ensuring that never empty value is returned if there is a duplicated
+        # property with non empty value
+        non_empty_props = {k for k, v in properties if v and v.strip()}
+        flattened = dict((k, v) for k, v in properties
```

Comment:
Nitpick: you can use `{k: v ...}` instead of `dict((k, v) ...)` here.

Reply:
You are right! Changed.